ui/pages/extension/AccessMesheryModal.js

```diff
@@ -0,0 +1,106 @@
+import React, {
+} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography, Grid, Button, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+const styles = makeStyles((theme) => ({
+  paper : {
+    position : 'absolute',
+    width : "30rem",
+    height : "20rem",
+    backgroundColor : theme.palette.background.paper,
+    boxShadow : theme.shadows[5],
+    borderRadius : "4px",
+    top : "0",
+    bottom : "0",
+    left : "0",
+    right : "0",
+    margin : "auto",
+    ["@media (max-width: 455px)"] : {
+      width : "100%"
+    },
+    zIndex : 5
+  },
+  header : {
+    display : "flex",
+    justifyContent : "center",
+    position : "absolute",
+    width : "100%",
+    top : "0.7rem",
+    fontWeight : "bold",
+    ["@media (max-width: 455px)"] : {
+      fontSize : "1rem"
+    },
+  },
+  imgWrapper : {
+    display : "flex",
+    justifyContent : "center"
+  },
+  headerWrapper : {
+    marginBottom : 12,
+    borderRadius : "4px 4px 0 0",
+    textAlign : 'center',
+    width : "100%",
+    padding : theme.spacing(1),
+    color : '#fff',
+    backgroundColor : "#252E31",
+    fontSize : "1rem",
+  },
+  closing : {
+    transform : "rotate(-90deg)",
+    "&:hover" : {
+      transform : "rotate(90deg)",
+      transition : "all .3s ease-in",
+      cursor : "pointer",
```

Comment:
Why is this not a standard part of the modal component?

Reply:
Now it is.